src/models/UsuarioSchema.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+import { ContaCreateSchema, ContaResponseSchema } from './ContaSchema';
+
+const requiredString = z.string().nonempty();
+const optionalString = z.string().nullable().optional();
+
+export const UsuarioCreateSchema = z.object({
+  nome: requiredString,
+  cpf: z.string().length(11),
+  telefone: z.string().length(11),
+
+  biografia: optionalString,
+  twitter: optionalString,
+  instagram: optionalString,
+  skoob: optionalString,
+  goodreads: optionalString,
+  fotoPerfil: optionalString,
+
+  conta: ContaCreateSchema,
+});
+
+export const UsuarioResponseSchema = UsuarioCreateSchema.extend({
+  id: z.number(),
+  conta: ContaResponseSchema.nullable().optional(),
+});
+
+export const UsuarioUpdateSchema = UsuarioCreateSchema.partial().extend({
+  id: z.number(),
+  conta: ContaResponseSchema.nullable().optional(),
+});
```

Comment:
como os dados que vem no Update são iguais aos do Response, o que acha de deixarmos assim?
```suggestion
export const UsuarioUpdateSchema = UsuarioResponseSchema;
```